### PR TITLE
Dat 4332 redirect to oasis search page on desktop view

### PR DESCRIPTION
**Pull request by idradm**
## Links

* http://wikia-inc.atlassian.net/browse/DAT-4332

## Description

Redirects to oasis search page while on desktop view.

## Reviewers

@Wikia/west-wing

### CODE REVIEW

File: front/main/app/routes/application.js [hunk 1]
```diff
@@ -212,7 +212,11 @@ export default Route.extend(
 			 * @param {string} query
 			 */
 			goToSearchResults(query) {
-				this.transitionTo('search', {queryParams: {query}});
+				if (Ember.$(window).width() >= 1064) {
```
Comment (rogatty) [inline front/main/app/routes/application.js]: It's fine for a quick fix but how about using https://github.com/freshbooks/ember-responsive to make it more sustainable?

File: front/main/app/breakpoints.js [hunk 2]
```diff
@@ -0,0 +1,7 @@
+/**
+ * Keep in sync with $mobile-range and $desktop-range from _settings.scss
```
Comment (rogatty) [inline front/main/app/breakpoints.js]: Can we have counterpart comment in `_settings.scss`?

File: front/main/app/mixins/responsive.js [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+const {Mixin, inject} = Ember;
+
+/**
+ * Mixin wrapping functionality of ui-responsive Ember addon to keep it's
+ * customization in the one place.
+ *
+ * Addon allows to check on which device - mobile or desktop user currently is,
+ * viewport, screen and body sizes.
+ * Example usage:
+ * this.get('responsive.isMobile')
```
Comment (idradm) [inline front/main/app/mixins/responsive.js]: Doc link would be useful here.

**Comment by idradm** [comment]
LGTM

**Comment by rogatty** [comment]
LGTM
May we expect some blog post also? ;)

**Comment by dianafa** [comment]
sure, blog post in my todoist ;)

**Comment by rybmat** [comment]
changing base branch, https://github.com/Wikia/mercury/pull/2511

**Pull request closed without merging**